### ISSUE TITLE
(Minor) Hardcore mode and tied-up cuffs

### ISSUE THREAD

**Issue by BreadDain** [IHateMyKite/UnforgivingDevices#16]
Description: getting tied up by cuffs does not preselect them for a hotkey use, which makes it completely impossible to get free without cheating if you didn't struggle from them prior being tied up.

Version of skyrim: SE
Version of mod: 2.0b9 commit 84d5fb1
Used patches: DD+BRRF, DCL, LBS

**Comment by BreadDain** [comment]
Just retried it, Tween menu works as expected, could select the device and open its WH menu. Map and journal works alright.

Unexpected behaviour: removes all interaction menus even if object is interactable, like NPC dialogue or item harvest for example.

**Comment by IHateMyKite** [comment]
Will test it. If its true, I will just have to add another bypass where pressing the activate key while looking on object will activate it.

**Comment by BreadDain** [comment]
Interface tooltip would completely disappear. It might be by mod setup ablame, so needs verification.

Basically, when you aim at NPC to talk or at flora to harvest it should show tooltip [E] Harvest X or Talk To. Which is missing when hardcore mode is in effect and hands are bound.

**Comment by BreadDain** [comment]
Yes, I have Better Third Person Selection, it changes the way how tooltip work, so probably should be listed as incompatibility. Closed.
